lib/signs_ui_web/controllers/signs_controller.ex

```diff
@@ -108,6 +57,10 @@ defmodule SignsUiWeb.SignsController do
   end
 
   defp do_params_to_signs({sign_id, sign}, enabled_map) do
-    {sign_id, %{sign | enabled?: Map.get(enabled_map, sign_id)}}
```

Comment:
What was this change for?

Reply:
All the signs that weren't on the individual route page were getting `nil` set for this value, since we're still passing all the signs to the controller.